assets/javascripts/discourse/pre-initializers/rich-editor.extension.js

```diff
@@ -0,0 +1,136 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import I18n, { i18n } from "discourse-i18n";
+import guessDateFormat from "discourse/plugins/discourse-calendar/discourse/lib/guess-best-date-format";
+
+const EVENT_ATTRIBUTES = {
+  name: { default: null },
+  start: { default: null },
+  end: { default: null },
+  reminders: { default: null },
+  minimal: { default: null },
+  closed: { default: null },
+  status: { default: "public" },
+  timezone: { default: "UTC" },
+  allowedGroups: { default: null },
+};
+
+/** @type {RichEditorExtension} */
+const extension = {
+  nodeSpec: {
+    event: {
+      attrs: EVENT_ATTRIBUTES,
+      group: "block",
+      content: "block+",
```

Comment:
we only need to set a `content` when we want this node to have children, this would define what children are allowed
```suggestion
```